clang/test/CodeGen/rtsan_insert_at_exit.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -fsanitize=realtime -emit-llvm -o - %s | FileCheck %s
+
+// __rtsan_realtime_exit should be inserted at all function returns
+
+int bar(int* x) [[clang::nonblocking]] {
+  return *x;
+}
+// CHECK: call{{.*}}__rtsan_realtime_exit
+// CHECK-NEXT: ret
```

Comment:
I was not able to figure out how to induce a multiple return statement in clang-IR. I may have to try to run a much more complex file through it. 

That would be an additional test, but this one is good to start us out.